components/snowflake/sources/common-delete.mjs

```diff
@@ -0,0 +1,83 @@
+import snowflake from "../snowflake.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import { v4 as uuid } from "uuid";
+
+export default {
+  type: "source",
+  props: {
+    snowflake,
+    db: "$.service.db",
+    timer: {
+      description: "Monitor for changes on this schedule",
```

Comment:
```suggestion
      description: "Monitor for changes on this schedule.",
```

Reply:
Added a period at the end of the sentence for consistency with Google's Developer docs style guide.